README.md

```diff
@@ -22,15 +22,18 @@ The application uses the following environment variables for configuration:
 10. `REVIEW_END_POINT`: This environment variable specifies the URL of the endpoint where the reviews are sent.
 11. `REVIEW_BOT_USER_TOKEN`: This environment variable represents the token used for authorization when sending the reviews.
 12. `WORKFLOW_FILE_PATH`: The path to your GitHub Actions workflow file. Default value is `.github/workflows/ci.js.yml`. Update this if you use a different path or file name for your workflow.
-13. `SKIP_GRADING`: If set to `true`, the action will only create a feedback in the LMS and not send a review to the review endpoint. Default value is `false`.
+13. `SKIP_GRADING`: If set to `true`, the action will only create a feedback in the LMS and not send a review to the review endpoint. Default value is `false`. Supported only in v1 of the action.
 
-> Note: You need to specify USER_PROMPT and ROLE_PROMPT mandatorily unless you provide a SYSTEM_PROMPT.
+> [!NOTE]
+> You need to specify USER_PROMPT and ROLE_PROMPT mandatorily unless you provide a SYSTEM_PROMPT.
+> V2 of the action does not require `SKIP_GRADING`. It will choose appropriate action based on the prompt provided.
```

Comment:
Remove the mention of `SKIP_GRADING` - you don't need to document older versions in the `main` branch - that's handled through the tags.